packages/terra-icon/src/icon/IconVolumeSetDecrease.jsx

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable */
```

Comment:
Seems like a bad practice to disable eslint entirely. I'd prefer leaving it on and opting out of _some_ rules only if they're really a problem. Tell me: why turn off linting?

Reply:
@jeremyfuksa told me these files are generated, so n/m: low value in linting them.

Reply:
Aw I had a detailed comment typed up and everything. I should refresh more often. Oh well, no sense letting documentation and context go to waste:
These files are generated automatically by [this script](https://github.com/cerner/terra-core/blob/main/packages/terra-icon/scripts/src/generate-icon/renderJsx.js) using these templates: [template.txt](https://github.com/cerner/terra-core/blob/main/packages/terra-icon/scripts/src/generate-icon/template.txt), [staticIconTemplate.txt](https://github.com/cerner/terra-core/blob/main/packages/terra-icon/scripts/src/generate-icon/staticIconTemplate.txt). [All of the icons](https://github.com/cerner/terra-core/tree/main/packages/terra-icon/src/icon) are this way.
Without this, it flags tons of silly deviations from our linting rules that can vary depending on the raw SVG transpiled into the templates from [one-cerner-style-icons](https://github.com/cerner/one-cerner-style-icons/tree/main/src/icons). I 💯 agree that it's bad practice, especially since it could let just any ol' code through from the raw SVGs. However, this would require changing the templates to meet terra's linting standards, ensuring all the SVG in the OCS icons transpiles correctly, and re-generating the jsx files for all of the icons, which seems like it's a separate issue and beyond the scope of this PR. Backlog?